File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportJava/Model/src/main/java/transport/model/Seat.java
package transport.model;

public class Seat extends Entity<Long>{
    private int number;
    private Long reservationId;
    private Long tripId;
    private String clientName;

    public Seat(Long id, int number, Long reservationId, Long tripId, String clientName) {
        super(id);
        this.number = number;
        this.reservationId = reservationId;
        this.tripId = tripId;
        this.clientName = clientName;
    }


    public int getNumber() {
        return number;
    }

    public void setNumber(int number) {
        this.number = number;
    }

    public Long getReservationId() {
        return  reservationId;
    }

    public void setReservationId(Long reservationId) {
        this.reservationId = reservationId;
    }

    public Long getTripId() {
        return tripId;
    }

    public void setTripId(Long tripId) {
        this.tripId = tripId;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }

    @Override
    public String toString() {
        return "Seat{" +
                "id=" + getId() +
                ", number=" + number +
                ", reservationId=" + reservationId +
                ", tripId=" + tripId +
                ", clientName='" + clientName + '\'' +
                '}';
    }
}

File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportC#/Model/Seat.cs
namespace FirmaTransportC_.Model
{
    public class Seat : Entity<long>
    {
        private int number;
        private long? reservationId;
        private long tripId;
        private string? clientName;
        public Seat(long id, int number, long? reservationId, long tripId, string? clientName) : base(id)
        {
            this.number = number;
            this.reservationId = reservationId;
            this.tripId = tripId;
            this.clientName = clientName;
        }

        public int Number
        {
            get { return number; }
            set { number = value; }
        }

        public long? ReservationId
        {
            get { return reservationId; }
            set { reservationId = value; }
        }

        public long TripId
        {
            get { return tripId; }
            set { tripId = value; }
        }

        public string? ClientName
        {
            get { return clientName; }
            set { clientName = value; }
        }

        public override string ToString()
        {
            return "Seat{" +
                "id=" + Id +
                ", number=" + number +
                ", reservationId=" + reservationId +
                ", tripId=" + tripId +
                '}';
        }
    }
}